projects/mat-table-filter/src/lib/mat-table-filter.service.ts

```diff
@@ -13,12 +13,26 @@ export class MatTableFilterService {
       const exampleColumn = exampleEntityObjectKeys[i];
       const exampleColumnValue = exampleEntity[exampleColumn];
       const itemColumnValue = item[exampleColumn];
-      if (!exampleColumnValue || exampleColumnValue instanceof Array) {
+      if (!exampleColumnValue && typeof exampleColumnValue !== 'boolean') {
```

Comment:
I'll approve this part and I have to override the rest I'm afraid. Thanks again.

Reply:
If you're taking the boolean fix you'll want [this line](https://github.com/HalitTalha/mat-table-extensions/pull/16/files#diff-3d4f8d536352779b2c0ccd3f9ad818b0R35) too. Glad you found a better implementation with lodash! I'm looking forward to seeing those features 👍